Better error message for "Upgrade Config from C26"

Rather than putting up with this error because their config is out of date:
```
[ERROR] [04/19/2017 18:14:32.636] [cromwell-system-akka.actor.default-dispatcher-4] [akka://cromwell-system/user/cromwell-service/ServiceRegistryActor/MetadataService] Error summarizing metadata
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '"SUMMARY_STATUS_ENTRY" where ("SUMMARY_TABLE_NAME" = 'WORKFLOW_METADATA_SUMMARY_' at line 1
```

We should have a nice error message that tells people what to do about it

--- a/services/src/main/scala/cromwell/services/ServicesStore.scala
+++ b/services/src/main/scala/cromwell/services/ServicesStore.scala
@@ -5,7 +5,6 @@ import cromwell.database.migration.liquibase.LiquibaseUtils
 import cromwell.database.slick.SlickDatabase
 import cromwell.database.sql.SqlDatabase
 import net.ceedubs.ficus.Ficus._
-import org.slf4j.LoggerFactory
 
 trait ServicesStore {
   def databaseInterface: SqlDatabase
@@ -30,20 +29,45 @@ trait SingletonServicesStore extends ServicesStore {
 
 object SingletonServicesStore {
 
-  private lazy val log = LoggerFactory.getLogger("SingletonServicesStore")
-  private val databaseConfig = {
-    val config = ConfigFactory.load.getConfig("database")
-    if (config.hasPath("config")) {
-      log.warn(
-        """
-          |Use of configuration path 'database.config' is deprecated.
-          |
-          |Move the configuration directly under the 'database' element, and remove the key 'database.config'.
-          |""".stripMargin)
-      config.getConfig(config.getString("config"))
-    } else {
-      config
-    }
+  private val databaseConfig = ConfigFactory.load.getConfig("database")
+
+  if (databaseConfig.hasPath("config")) {
+      val msg = """
+        |*******************************
+        |***** DEPRECATION MESSAGE *****
+        |*******************************
+        |
+        |Use of configuration path 'database.config' has been deprecated.
+        |
+        |Move the configuration directly under the 'database' element, and remove the key 'database.config'.
+        |
+        |""".stripMargin
+    throw new Exception(msg)
+  } else if (databaseConfig.hasPath("driver")) {
+    val msg =
+      """
+      |*******************************
+      |***** DEPRECATION MESSAGE *****
+      |*******************************
+      |
+      |Use of configuration path 'database.driver' has been deprecated. Replace with a "profile" element instead, e.g:
+      |
+      |database {
+      |  #driver = "slick.driver.MySQLDriver$" #old
+      |  profile = "slick.jdbc.MySQLProfile$"  #new
+      |  db {
+      |    driver = "com.mysql.jdbc.Driver"
+      |    url = "jdbc:mysql://host/cromwell?rewriteBatchedStatements=true"
+      |    user = "user"
+      |    password = "pass"
+      |    connectionTimeout = 5000
+      |  }
+      |}
+      |
+      |Cromwell thanks you.
+      |""".stripMargin
+    throw
+      new Exception(msg)
   }
 
   import ServicesStore.EnhancedSqlDatabase

